com.unity.multiplayer.mlapi/Runtime/Core/NetworkUpdateLoopSystem.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using UnityEngine;
+
+
+namespace MLAPI
+{
+    /// <summary>
+    ///  NetworkUpdateLoopBehaviour
+    ///  Derive from this class if you need to register a NetworkedBehaviour based class
+    /// </summary>
+    public class NetworkUpdateLoopBehaviour:NetworkedBehaviour,INetworkUpdateLoopSystem
```

Comment:
spaces? :)
`NetworkUpdateLoopBehaviour : NetworkedBehaviour, INetworkUpdateLoopSystem`

Reply:
Will make that adjustment.